### ISSUE TITLE
Prettier plugins are not being used?

### ISSUE THREAD

**Issue by pricecurrent** [hyrious/prettierd#4]
Sublime Text 4, MacOS 

In my project there is a prettier plugin being used `prettier-plugin-tailwindcss` — basically it sorts html classes in a certain way.

When I save a file in sublime and prettierd does it's job, the `prettier-plugin-tailwind` is not found/executed — classes are not being sorted.

If I run however 
```cat src/components/MySettingsOverlay/ShowMySettingsButton.vue | prettierd src/components/MySettingsOverlay/ShowMySettingsButton.vue``` 

in the terminal output I can see the plugin being used.

**Comment by hyrious** [comment]
Yes, because this plugin loads prettier [from the global path](https://github.com/hyrious/prettierd/blob/5ddb6b30957649fa6c04cc8371cfc71cdc296d84/prettierd.mjs#L55-L59), it can only find plugins that are installed globally too -- Try `npm i -g prettier-plugin-tailwindcss` then restart sublime text.

If the plugin can find prettier from the local project, it maybe then can recognize local plugins too. I'm not in a hurry to implement that since it suits my need already.

Feel free to make a PR if you would like to contribute to this plugin!
